bucket/ccl.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.11.5",
+    "homepage": "https://ccl.clozure.com/",
+    "description": "Clozure Common Lisp",
+    "license": "Apache-2.0",
+    "url": "https://github.com/Clozure/ccl/releases/download/v1.11.5/ccl-1.11.5-windowsx86.zip",
+    "hash": "da5d8830e38742a5e76dc09a5fdebf8cb8a09a6fee87d7919ffd67ae2584cdd4",
+    "extract_dir": "ccl",
+    "env_add_path": "scripts",
```

Comment:
>  In the scripts/ directory of the ccl directory, there are two files named ccl and ccl64. Copy these files into /usr/local/bin or some other directory that is on your path, and then edit them so that the value of CCL_DEFAULT_DIRECTORY is your ccl directory. You can then start up the lisp by typing ccl or ccl64.

If there are only 2 files, why not add them as bins?

Reply:
Or will this folder grow in time?

Reply:
As far as I know, more files may be added in the future, like for example, right now ther are also some shell scripts.

Reply:
Ok, I'll add them as bin

Reply:
Thank you so much.